base/bucket_gocb.go

```diff
@@ -1360,6 +1360,53 @@ func (bucket *CouchbaseBucketGoCB) DeleteWithXattr(k string, xattrKey string) er
 
 }
 
+func (bucket *CouchbaseBucketGoCB) GetXattr(k string, xattrKey string, xv interface{}) (casOut uint64, err error) {
+
+	worker := func() (shouldRetry bool, err error, value interface{}) {
+		res, lookupErr := bucket.Bucket.LookupInEx(k, gocb.SubdocDocFlagAccessDeleted).
+			GetEx(xattrKey, gocb.SubdocFlagXattr).Execute()
+
+		switch lookupErr {
+		case nil:
+			res.Content(xattrKey, xv)
+			cas := uint64(res.Cas())
+			return false, err, cas
+		case gocbcore.ErrSubDocBadMulti:
+			xattrErr := res.Content(xattrKey, xv)
+			Debugf(KeyCRUD, "No xattr content found for key=%s, xattrKey=%s: %v", UD(k), UD(xattrKey), xattrErr)
+			cas := uint64(res.Cas())
+			return false, gocb.ErrSubDocBadMulti, cas
+		case gocbcore.ErrKeyNotFound:
+			Debugf(KeyCRUD, "No document found for key=%s", UD(k))
+			return false, gocb.ErrKeyNotFound, 0
+		case gocbcore.ErrSubDocMultiPathFailureDeleted, gocb.ErrSubDocSuccessDeleted:
```

Comment:
Similar to the above - I though MultiPathFailureDeleted can only be raised when issuing multiple GetEx in the subdoc operation - what permutation returns this error for the above subdoc operation?

Reply:
I did experience this when creating the tests, I will explicitly write one that hits it.

Reply:
As above.